bin/disco_autoscale.py

```diff
@@ -31,9 +31,12 @@ def parse_arguments():
 
     parser_delete_group = subparsers.add_parser('deletegroup', help='Delete autoscaling group')
     parser_delete_group.set_defaults(mode="deletegroup")
-    parser_delete_group.add_argument("--hostclass", required=True, help='Name of autoscaling group')
     parser_delete_group.add_argument("--force", action='store_true',
                                      required=False, default=False, help='Force deletion')
+    parser_delete_specifier_group = parser_delete_group.add_mutually_exclusive_group(required=True)
+    parser_delete_specifier_group.add_argument("--hostclass", default=None, help='Name of the hostclass')
```

Comment:
It doesn't make sense to specify both a hostclass and a group name, so `parser_delete_specifier_group` is a mutually exclusive group where you can only specify either hostclass or group name.

https://docs.python.org/2/library/argparse.html#mutual-exclusion